### PR TITLE
Add CI and introduce minor pep8 fixes

### PR DESCRIPTION
**Pull request by smn**
Some minor pep8 fixes done while I was reading the code.
Some of these files don't seen runnable though, especially `watch.py`.

The code base seems to default to using tabs instead of spaces, I've added a `setup.cfg` that instructs flake8 to ignore any violations around the use of tabs.

Travis CI is passing with these changes: https://travis-ci.org/smn/pool/builds/329386972

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,15 @@
+language: python
+python:
+- '2.7'
```
Comment (SeiryuZ) [inline .travis.yml]: I'm new so I shouldn't be the one who has the final say. 

However, I would strongly suggests that we target the python version that the hosts machine will use, no need to bother with the other versions to keep it simple

**Review by SeiryuZ — COMMENTED** [review]
This PR looks good to me, however, since I'm new I won't do anything yet. 

In general, I also would like to do some more cleaning up on the code

File: .travis.yml (hunk 1 above)
Comment (SeiryuZ) [reply .travis.yml]: Since the readme says ``pip3`` I'd say we should target python 3 only  (thanks for the hint matheusb.comp)

**Comment by smn** [comment]
Added minor code syntax, variable name & import fixes for @matheusb-comp 's last commit.

File: .travis.yml (hunk 1 above)
Comment (smn) [reply .travis.yml]: thanks, updated.

**Comment by matheusb-comp** [comment]
Wow, thanks for the help! I'll just check a few things and merge.

**Comment by smn** [comment]
happy to help. I was wanting to write some tests, since those seem to be lacking a bit. Any suggestions on where to start would be appreciated.

**Comment by smn** [comment]
Travis seems to be suffering a bit of an outage so the CI won't be reporting much yet: https://www.traviscistatus.com/incidents/c1s4dlyxd4lf

**Comment by matheusb-comp** [comment]
Thanks!

**Pull request merged**